.travis.yml

```diff
@@ -49,8 +51,8 @@ script:
       make get-deps;
     fi;
     if [ "$TRAVIS_GO_VERSION" == "tip" ] ||
-        [ "$TRAVIS_GO_VERSION" == "1.13.x" ] ||
-        [ "$TRAVIS_GO_VERSION" == "1.12.x" ]; then
+        [ "$TRAVIS_GO_VERSION" == "1.14.x" ] ||
+        [ "$TRAVIS_GO_VERSION" == "1.13.x" ]; then
 
       if [ "$TRAVIS_OS_NAME" = "windows" ]; then
         make unit-no-verify;
```

Comment:
Do we still want this?

Reply:
yeah need this for now because the verification step is a bash script that won't work on windows. would need to rewrite it to be compatible with windows.